pkg/virtualnetwork/mux.go

```diff
@@ -26,7 +27,7 @@ func (n *VirtualNetwork) Mux() http.Handler {
 	mux.HandleFunc("/leases", func(w http.ResponseWriter, r *http.Request) {
 		_ = json.NewEncoder(w).Encode(n.networkSwitch.IPs.Leases())
 	})
-	mux.HandleFunc("/connect", func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc(types.ConnectPath, func(w http.ResponseWriter, r *http.Request) {
```

Comment:
How are we going to filter using this endpoint, like only allow requests with url == "/connect"?

Reply:
Yes the idea is to restrict what is visible for vsock a little bit.